ledger/participant-state/kvutils/src/main/scala/com/daml/ledger/participant/state/kvutils/Conversions.scala

```diff
@@ -215,4 +217,19 @@ private[state] object Conversions {
       )
     )
 
+  def encodeBlindingInfo(blindingInfo: BlindingInfo): DamlTransactionBlindingInfo =
+    DamlTransactionBlindingInfo.newBuilder
+      .addAllDisclosure(blindingInfo.disclosure.map { disclosureEntry =>
+        DamlTransactionBlindingInfo.DisclosureEntry.newBuilder
+          .setNodeId(disclosureEntry._1.index.toString)
+          .addAllDisclosedTo(disclosureEntry._2.asInstanceOf[Set[String]].asJava)
```

Comment:
Are `DamlStateValue`s included in the hash for the consensus check? If so, we need to sort this set.

Reply:
It will be persisted so I believe we do. Thanks!

Reply:
Fixed.